wallet/src/de/schildbach/wallet/ui/CreateUsernameActivity.kt

```diff
@@ -114,15 +115,29 @@ class CreateUsernameActivity : InteractionAwareActivity() {
         val username = intent?.extras?.getString(EXTRA_USERNAME)
         val invite = intent.getParcelableExtra<InvitationLinkData>(EXTRA_INVITE)
         val fromOnboardng = intent.getBooleanExtra(EXTRA_FROM_ONBOARDING, false)
-        val createUsernameArgs = CreateUsernameArgs(
-            actions = action,
-            userName = username,
-            invite = invite,
-            fromOnboardng = fromOnboardng,
-        )
-        val bundle = Bundle()
-        bundle.putParcelable(CreateUsernameFragment.CREATE_USER_NAME_ARGS, createUsernameArgs)
-
-        navController.setGraph(navController.graph, bundle)
+
+        lifecycleScope.launchWhenCreated {
+            val navHostFragment = supportFragmentManager.findFragmentById(R.id.nav_host_create_user_name_fragment) as NavHostFragment
+            val navController = navHostFragment.navController
```

Comment:
This class has `private val navController by lazy` which handles finding the navController. We can either user that or remove it if it's not needed.